File: python/1037-intervalo.py
n = float(input())

if n > 100 or n < 0:
    print("Fora de intervalo")
elif n >= 0 and n <= 25 :
    print("Intervalo [0,25]")
elif n > 25 and n <= 50:
    print("Intervalo (25,50]")
elif n > 50 and n <= 100:
    print("Intervalo (75,100]")

File: kotlin/1037-intervalo.kt
import java.util.*

fun main(args: Array<String>) {

  var N = readLine()!!.toDouble()

  if (N > 100 || N < 0) {
    println("Fora de intervalo")
  } else if (N >= 0 && N <= 25) {
    println("Intervalo [0,25]")
  } else if (N > 25 && N <= 50) {
    println("Intervalo (25,50]")
  } else if (N > 50 && N <= 75) {
    println("Intervalo (50,75]")
  } else if (N > 75 && N <= 100) {
    println("Intervalo (75,100]")
  }
	
}
